eng/Version.Details.xml

```diff
@@ -22,44 +22,20 @@
     </Dependency>
-    <Dependency Name="ILLink.Tasks" Version="0.1.6-prerelease.19380.1" CoherentParentDependency="Microsoft.Dotnet.Toolset.Internal">
+    <Dependency Name="ILLink.Tasks" Version="0.1.6-prerelease.20152.3" CoherentParentDependency="Microsoft.Dotnet.Toolset.Internal">
       <Uri>https://github.com/mono/linker</Uri>
-      <Sha>1127689f262d52ea8ff68ef03d706fa62b3b40a1</Sha>
+      <Sha>a9bfc450a9da6b050ba043ca43b9241beb6f3f60</Sha>
       <RepoName>linker</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.1.2-servicing.20066.2" CoherentParentDependency="Microsoft.Private.CoreFx.NETCoreApp">
-      <Uri>https://github.com/dotnet/coreclr</Uri>
-      <Sha>fd22fd600a2526a7c58536da0e80325eb710d7f1</Sha>
-      <RepoName>coreclr</RepoName>
+    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="5.0.0-preview2.20160.6">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>d12f79a4d1b908636b96005fe30f547e54cea3ee</Sha>
+      <RepoName>runtime</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.1.2-servicing.20066.2" CoherentParentDependency="Microsoft.Private.CoreFx.NETCoreApp">
-      <Uri>https://github.com/dotnet/coreclr</Uri>
-      <Sha>fd22fd600a2526a7c58536da0e80325eb710d7f1</Sha>
-      <RepoName>coreclr-portable</RepoName>
-    </Dependency>
-    <Dependency Name="NETStandard.Library" Version="2.1.0" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
-      <Uri>https://github.com/dotnet/standard</Uri>
-      <Sha>a5b5f2e1e369972c8ff1e2183979fab6099f52ef</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.7.0-servicing.20067.2" CoherentParentDependency="Microsoft.NETCore.App">
-      <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>e946cebe43a510e8c6476bbc8185d1445df33a1a</Sha>
-      <RepoName>corefx</RepoName>
-    </Dependency>
-    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.7.0-servicing.20067.2" CoherentParentDependency="Microsoft.NETCore.App">
-      <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>e946cebe43a510e8c6476bbc8185d1445df33a1a</Sha>
-      <RepoName>corefx-portable</RepoName>
-    </Dependency>
-    <Dependency Name="Microsoft.NETCore.App" Version="3.1.2-servicing.20067.4" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
-      <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>916b5cba268e1e1e803243004f4276cf40b2dda8</Sha>
-      <RepoName>core-setup</RepoName>
-    </Dependency>
-    <Dependency Name="Microsoft.NETCore.App" Version="3.1.2-servicing.20067.4" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
-      <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>916b5cba268e1e1e803243004f4276cf40b2dda8</Sha>
-      <RepoName>core-setup-portable</RepoName>
+    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="5.0.0-preview2.20160.6">
```

Comment:
I don't think runtime produces this package now since consolidation. I'd pick `Microsoft.NETCore.App.Runtime.win-x64`, it's usual to pick the runtime pack when depending on a sharedfx-producing repo.

Reply:
Fixed.